doc/docusaurus/docs/auction-smart-contract/end-to-end/mint.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 15
+---
+
+# Minting the Token to be Auctioned
+
+Before we can start the auction, we need to mint a token to be auctioned.
+To do so, we first must determine the token's currency symbol and name.
+To mint or burn tokens with a specific currency symbol (`currencySymbol`), a Plutus script whose hash matches `currencySymbol` must be provided, and is used to validate the minting or burning action.
+Therefore, we'll first write the on-chain script, then compute its hash and use it as the currency symbol.
+
+## On-chain Minting Policy Script
+
+The full minting policy code can be found at [AuctionMintingPolicy.hs](https://github.com/IntersectMBO/plutus-tx-template/blob/main/src/AuctionMintingPolicy.hs).
+The main logic is in the following function:
+
+<LiteralInclude file="AuctionMintingPolicy.hs" language="haskell" title="AuctionMintingPolicy.hs" start="-- BLOCK1" end="-- BLOCK2" />
+
+This script will pass if the following two conditions are met:
+
+1. The transaction is signed by a specific public key.
+2. The transaction mints exactly one token, whose currency symbol matches the script's hash (i.e., `ownCurrencySymbol ctx`).
+  The token name can be anything.
+
+> :pushpin: **NOTE**
+>
+> A token minted in this way is _not_ considered a non-fungible token (NFT) because, while only one token can be minted in a single transaction, multiple transactions can mint additional tokens with the same currency symbol and token name.
+> To create a truly unique token, you would need a more complex minting policy, but for simplicity, that is not covered here.
+
+## Compile and Generate Blueprint for the Minting Policy
+
+Next, we need to compile the minting policy script and create its blueprint.
+To do so, we first need to supply a public key hash, which the minting policy will use for checking condition 1 above.
+Assuming the seller is the one minting the token, this should be the seller's public key hash.
+Open `GenMintingPolicyBlueprint.hs` in the `on-chain` directory, and replace `error "Replace with seller pkh"` with the content of `off-chain/seller.pkh`.
```

Comment:
Is there a reason to put auction validator script and minting policy script in two different blueprint files?
I can see how two different plutus scripts (validator, policy) together constitute a one logical "contract".

Reply:
To generate blueprint for the auction validator, we need to know the hash of the minting validator, and currently the best way to get the hash is from the blueprint.